common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -152,11 +153,10 @@ def B(*args, **kwargs):
     AUTH_ENTRY_PROFILE,
     AUTH_ENTRY_REGISTER,
 
-    # TODO (ECOM-369): For the A/B test of the combined
-    # login/registration, we needed to introduce two
-    # additional end-points.  Once the test completes,
-    # delete these constants from the choices list.
-    # pylint: disable=fixme
+    # This is left-over from an A/B test
+    # of the new combined login/registration page (ECOM-369)
```

Comment:
That seems like a good idea. I've created [ECOM-1017](https://openedx.atlassian.net/browse/ECOM-1017).